onadata/apps/logger/management/commands/populate_submission_counters.py

```diff
@@ -153,24 +153,24 @@ def handle(self, *args, **kwargs):
                     elif verbosity >= 2:
                         self.stdout.write(f'\tNo daily counters data...')
 
-                for key, total in total_submissions.items():
-                    year, month = key.split('-')
-                    monthly_counters.append(MonthlyXFormSubmissionCounter(
-                        year=year,
-                        month=month,
-                        xform_id=xf.pk,
-                        user_id=user.pk,
-                        counter=total,
-                    ))
-
-                if monthly_counters:
-                    if verbosity >= 2:
-                        self.stdout.write(f'\tInserting monthly counters data...')
-                    MonthlyXFormSubmissionCounter.objects.bulk_create(
-                        monthly_counters, batch_size=chunks
-                    )
-                elif verbosity >= 2:
-                    self.stdout.write(f'\tNo monthly counters data!')
+                    for key, total in total_submissions.items():
+                        year, month = key.split('-')
+                        monthly_counters.append(MonthlyXFormSubmissionCounter(
+                            year=year,
+                            month=month,
+                            xform_id=xf.pk,
+                            user_id=user.pk,
+                            counter=total,
+                        ))
+
+                    if monthly_counters:
+                        if verbosity >= 2:
+                            self.stdout.write(f'\tInserting monthly counters data...')
+                        MonthlyXFormSubmissionCounter.objects.bulk_create(
+                            monthly_counters, batch_size=chunks
+                        )
+                    elif verbosity >= 2:
+                        self.stdout.write(f'\tNo monthly counters data!')
```

Comment:
I can see how this mistake happened. The nesting is pretty deep:
```
class
    def
        for
            with
                for
                    for  ## the one with mistaken indentation
```

I'd suggest refactoring this into a few different methods so that `handle()` isn't as difficult to maintain. What do you think?

Reply:
Totally agree with the refactoring :-)